defaultTheme.js

```diff
@@ -0,0 +1,13 @@
+import { DefaultTheme } from 'react-native-paper'
+
+const theme = {
+  ...DefaultTheme,
+  colors: {
+    ...DefaultTheme.colors,
+    primary: '#2165E3',
+    white: '#eee',
```

Comment:
this is not white, let's use a name that captures this better. note that paper already has existing names of colors we could override and use, and they would be semantically correct. see what's available here https://github.com/callstack/react-native-paper/blob/main/src/styles/DefaultTheme.tsx

Reply:
Typo 🤦